.github/ISSUE_TEMPLATE/10-issue_report.yaml

```
@@ -0,0 +1,136 @@
+name: Report a Bug
+title: "<title>"
+description: |
+  You found a bug in Bareos causing the application to crash or
+  throw an exception, a component is buggy, or something looks wrong.
+labels: needs triage
+body:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for using Bareos and taking the time to fill out this bug report!
+
+        If you are looking for support, please check out our documentation
+        or consider asking a question on mailing list:
+
+          - https://doc.bareos.org/
+          - https://groups.google.com/u/1/g/bareos-users
+  - type: markdown
+    attributes:
+      value: |
+        Before filling the form fields, please consider the following:
+        - Ensure that you have searched the [existing issues](https://github.com/bareos/bareos/issues)
+        - Read the [troubleshooting guide](https://docs.bareos.org/Appendix/Troubleshooting.html)
+        - Read the [guide to filing a bug](https://docs.bareos.org/Appendix/Troubleshooting.html#bug-reports)
+
+        **What is a bug?** A bug describe a unexpected trouble with the software which
+        prove a divergence with the documentation, and/or a crash (trace) that are
+        somewhat reproducible.
+
+        We always recommend to start discussing your issue on the mailing list.
```

Maybe we want to word that even stronger like:
```suggestion
        We recommend to discuss your issue on the mailing list first.
```